src/ctap/credential_id.rs

```diff
@@ -356,12 +353,7 @@ mod test {
 
         for length in (1..CBOR_CREDENTIAL_ID_SIZE).step_by(16) {
             assert_eq!(
-                decrypt_credential_id(
-                    &mut env,
-                    encrypted_id[..length].to_vec(),
-                    &rp_id_hash,
-                    false
-                ),
+                decrypt_credential_id(&mut env, encrypted_id[..length].to_vec(), &rp_id_hash,),
```

Comment:
nit: extra comma

Reply:
done, I wonder whether this can be caught by any linting check?

Reply:
We just need for the [trailing_comma](https://github.com/rust-lang/rustfmt/issues/3379) rustfmt config to stabilize I guess (and possibly use a more recent compiler if needed).